### ISSUE TITLE
[req]: torrentegg.com

### ISSUE THREAD

**Issue by mytelegrambot** [Jackett/Jackett#15625]
### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Is there already a request for your tracker?

- [X] I have checked older issues, open and closed

### Type

Public

### Tracker details

```markdown
- Name: torrentegg.com
- URL: https://torrentegg28.com/torrent/newest.html
- Description: [Korea] Public Torrent Tracker for Movies, TV...
- Additional Comments: 
  It's similar to https://torrentqq.com/
```

**Comment by garfield69** [comment]
Not only is it similar, it's exactly identical in layout and content.
You can plug in https://torrentegg28.com/ into the current torrentqq indexer and it works exactly the same.
I'm not inclined to create a separate indexer for an identical site.
I'm thinking of just adding the torrentegg domain to the alternate sitelinks of torrentqq.

**Comment by garfield69** [comment]
closed as a duplicate of #11208
added the torrentegg domain to the alternate sitelinks of torrentqq

**Comment by garfield69** [comment]
v0.22.716
